test/mpi/configure.ac

```diff
@@ -263,6 +263,8 @@ AC_ARG_WITH(dtpools-datatypes,
 dtp_args="--with-dtpools-datatypes=${with_dtpools_datatypes}"
 
 AC_CONFIG_COMMANDS([gentests],[$srcdir/maint/gentests_dtp.sh $dtp_args],[dtp_args="$dtp_args"])
+#Test collectives algorithms with CVARs
+AC_CONFIG_COMMANDS([collcvartests],[$srcdir/maint/test_coll_algos.sh > $srcdir/coll/testlist.cvar]
 
```

Comment:
* Missing closing parenthesis. 
* Don't use redirection here, use it inside `test_coll_algos.sh`. 
* Write to `coll/testlist.cvar`, which goes to `$builddir` rather than `$srcdir`.

Reply:
I addressed these.